jestSetup.js

```diff
@@ -0,0 +1,32 @@
+jest.mock('./index.js', () => ({
+  isAgentStarted: jest.fn(),
+  onNavigationStateChange: jest.fn(),
+  componentDidAppearListener:jest.fn(),
+  onStateChange:jest.fn(),
+  startAgent: jest.fn(),
+  getReactNativeVersion:jest.fn(),
+  analyticsEventEnabled: jest.fn(),
+  networkRequestEnabled: jest.fn(),
+  networkErrorRequestEnabled: jest.fn(),
+  httpRequestBodyCaptureEnabled: jest.fn(),
+  recordBreadcrumb: jest.fn(),
+  recordCustomEvent: jest.fn(),
+  crashNow: jest.fn(),
+  currentSessionId: jest.fn(),
+  noticeNetworkFailure: jest.fn(),
+  recordMetric: jest.fn(),
+  removeAllAttributes: jest.fn(),
+  recordError: jest.fn(),
+  setMaxEventBufferTime: jest.fn(),
+  setMaxEventPoolSize: jest.fn(),
+  startInteraction: jest.fn(),
+  endInteraction:jest.fn(),
+  setInteractionName:jest.fn(),
+  setAttribute: jest.fn(),
+  removeAttribute: jest.fn(),
+  incrementAttribute: jest.fn(),
+  setJSAppVersion: jest.fn(),
+  setUserId: jest.fn(),
+  recordError: jest.fn(),
```

Comment:
`recordError` is also on line 19

Reply:
Good catch, I'll clean that up for our next release.